packages/components/src/Link/docs/Link.stories.tsx

```diff
@@ -37,6 +38,12 @@ export default meta;
 
 type Story = StoryObj<typeof meta>;
 
+function GetColorScheme() {
```

Comment:
[Nit] Not sure I see the value of this function

Reply:
This is to prevent this error: 
React Hook "useColorSchemeContext" is called in function "render" that is neither a React function component nor a custom React Hook function.